code/datums/status_effects/debuffs.dm

```diff
@@ -458,7 +458,8 @@
 	var/old_health
 
 /datum/status_effect/kindle/tick()
-	owner.Knockdown(15)
+	owner.Knockdown(15, TRUE, FALSE, 15)
+	owner.Stun(15)
```

Comment:
This extra Stun() is redundant since you're already applying the hard stun via knockdown

Reply:
Glad to know I weren't wrong to start with, the above comment tenderly bamboozled me.